### PR TITLE
204 create validation reference for snowflake setup instructions

### PR DESCRIPTION
**Pull request by sungchun12**
#204

### CODE REVIEW

**Comment by sungchun12** [comment]
@leoebfolsom take a look with a keen eye on the schema permission screenshot.

File: docs/deployment_testing/team_cloud/getting_started_for_customers/data_sources/snowflake.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ It is best practice to create a separate role for the Datafold integration (e.g.
 
 ```sql
 CREATE ROLE DATAFOLDROLE;
+CREATE USER DATAFOLD;
```
Comment (leoebfolsom) [inline docs/deployment_testing/team_cloud/getting_started_for_customers/data_sources/snowflake.md:26]: Why create the user here? Isn't the user created on line 27 (`CREATE USER DATAFOLD DEFAULT_ROLE = "DATAFOLDROLE" MUST_CHANGE_PASSWORD = FALSE;`)?

**Review by leoebfolsom — APPROVED** [review]
Approved but please review/respond to my question about the `create user datafold;` line(s). Screenshots look awesome! 🦃

File: docs/deployment_testing/team_cloud/getting_started_for_customers/data_sources/snowflake.md (hunk 1 above)
Comment (sungchun12) [reply docs/deployment_testing/team_cloud/getting_started_for_customers/data_sources/snowflake.md]: You're right!

My reading eyes betrayed me. I removed this extra user creation command

**Pull request merged**